cli/util/uploader.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"crypto/sha256"
 	"fmt"
+	"slices"
```

Comment:
> cli: add check for empty OIDs in index file upload-bin command 

"cli: verify index file construction in `upload-bin`" or something like this. You may add additional notes in the commit body.